watertap/examples/flowsheets/generic_desalination_train/costing/stream_costing.py

```diff
@@ -0,0 +1,40 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
```

Comment:
Need to move all of these files out of watertap.examples, which no longer exists